### PR TITLE
Refactor dropdown menu for power actions (bsc#955810)

### PR DESCRIPTION
**Pull request by vwallfahrer**
- Dropdown menu collapse after action
- Shows flash message
- Shows blockUI while rebooting

### CODE REVIEW

File: crowbar_framework/app/views/nodes/_buttons_power.html.haml [hunk 1]
```diff
@@ -2,23 +2,28 @@
...
+      %ul.dropdown-menu{ "aria-labelledby" => "dropdownMenu1" }
         %li{ role: "presentation" }
-          = link_to t(".reboot"), hit_node_path(@node.handle, "reboot"), role: "menuitem", remote: true
+          %a{ :href => hit_node_path(@node.handle, "reboot"), role: "menuitem",
+            data: { blockui_click: t(".blockui_message") } }
+            = t(".reboot")
         %li{ role: "presentation" }
-          = link_to t(".shutdown"), hit_node_path(@node.handle, "shutdown"), role: "menuitem", remote: true
-        %li.divider{ role: "presentation" }
-      - if @node.bmc_set?
```
Comment (jsuchome) [inline crowbar_framework/app/views/nodes/_buttons_power.html.haml]: I think you've lost this condition. In the old code, Power On button could be shown only `if @node.bmc_set?`

**Comment by tboerger** [comment]
The other power buttons should get also a blockui message. Otherwise :+1:

File: crowbar_framework/app/views/nodes/_buttons_power.html.haml (hunk 1 above)
Comment (tboerger) [inline crowbar_framework/app/views/nodes/_buttons_power.html.haml]: True, I missed that

**Comment by rsalevsky** [comment]
Commit message is wrong... bsc#955810

**Comment by tboerger** [comment]
Closed in favor of https://github.com/crowbar/crowbar-core/pull/375

**Pull request closed without merging**